src/operations/txbuild.js

```diff
@@ -50,7 +50,7 @@ function fundTransaction(txB: bitcoinjs.TransactionBuilder, paymentAddress: stri
   const txFee = estimateTXBytes(txB, 0, 0) * feeRate
   const outAmounts = sumOutputValues(txB)
   const change = addUTXOsToFund(txB, utxos, txFee + outAmounts - inAmounts, feeRate)
-  txB.tx.outs[changeIndex].value += change
+  txB.__tx.outs[changeIndex].value += change
```

Comment:
The internal field `TransactionBuilder.tx` has been prefixed with `__tx` to indicate that it is private and not to be accessed by clients. I'm sure there are workarounds to achieve the same result without accessing this field directly but I didn't spend the time to figure that out as the tests pass.